api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -64,7 +64,7 @@ public void validate(Object object, Errors errors) {
 		}
 		
 		if (object == null) {
-			throw new IllegalArgumentException("The personAddress object should not be null");
+			errors.reject("error.Address");
```

Comment:
according to the needs of your ticket,i see you need to use some logging and cannot see it here